mlir/test/Dialect/ArmSME/tile-allocation-invalid.mlir

```diff
@@ -1,19 +1,15 @@
-// RUN: mlir-opt %s -allocate-arm-sme-tiles -split-input-file -verify-diagnostics
+// RUN: mlir-opt %s -convert-scf-to-cf -test-arm-sme-tile-allocation -verify-diagnostics
```

Comment:
I agree that having tests in CF too would be good (since this is an algorithm on CFGs). I've used SCF in these tests as it's easier to write/read (but cases like this one, which is just a branch, can easily be written in CF).